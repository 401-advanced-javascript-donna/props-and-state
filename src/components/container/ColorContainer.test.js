import React from 'react';
import { shallow } from 'enzyme';
import ColorContainer from './ColorContainer';

describe('Color Container component', () => {
  it('should render color container', () => {
    const wrapper = shallow(<ColorContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
