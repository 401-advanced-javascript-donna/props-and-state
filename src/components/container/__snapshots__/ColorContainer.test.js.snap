// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Container component should render color container 1`] = `
<Fragment>
  <ColorDisplay
    backgroundColor="black"
  />
  <ColorPicker
    colors={
      Array [
        "red",
        "green",
        "black",
        "yellow",
      ]
    }
    selectColor={[Function]}
  />
</Fragment>
`;
